folio-spring-cql/src/main/java/org/folio/spring/cql/JpaCqlRepository.java

```diff
@@ -10,5 +10,9 @@ public interface JpaCqlRepository<T, I> extends JpaRepository<T, I> {
 
   Page<T> findByCql(String cql, Pageable pageable);
 
+  Page<T> findByCqlIgnoreCase(String cql, Pageable pageable, boolean ignoreCase);
+
   long count(String cql);
+
+  long countIgnoreCase(String cql, boolean ignoreCase);
```

Comment:
For me if there is a prefix `IgnoreCase` then there is not need for `ignoreCase` parameter. Because if it exists then we have like method duplicates: `findByCqlIgnoreCase` with ignoreCase = false duplicate `findByCql` method.